flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/PipelinedSubpartition.java

```diff
@@ -44,15 +44,15 @@
  * {@link PipelinedSubpartitionView#notifyDataAvailable() notify} a read view created via
  * {@link #createReadView(BufferAvailabilityListener)} of new data availability. Except by calling
  * {@link #flush()} explicitly, we always only notify when the first finished buffer turns up and
- * then, the reader has to drain the buffers via {@link #pollBuffer()} until its return value shows
- * no more buffers being available. This results in a buffer queue which is either empty or has an
- * unfinished {@link BufferConsumer} left from which the notifications will eventually start again.
+ * then, the reader has to drain the buffers via {@link #pollBuffer(boolean)} until its return value
+ * shows no more buffers being available. This results in a buffer queue which is either empty or has
+ * an unfinished {@link BufferConsumer} left from which the notifications will eventually start again.
  *
  * <p>Explicit calls to {@link #flush()} will force this
  * {@link PipelinedSubpartitionView#notifyDataAvailable() notification} for any
  * {@link BufferConsumer} present in the queue.
  */
-class PipelinedSubpartition extends ResultSubpartition {
+public class PipelinedSubpartition extends ResultSubpartition {
```

Comment:
is it necessary change?

Reply:
The change is for test.

Reply:
Refer to this https://github.com/apache/flink/pull/10375/commits/2f538ab20262fbe9981d4851b93e9bee04fc6ad0#r354104779